pkg/utils/printer/printer_test.go

```diff
@@ -166,7 +166,7 @@ func TestPrinterResultJsonRequestId(t *testing.T) {
 			},
 		},
 	}
-	res.ApiResponse.Header.Add("location", "https://api.ionos.com/cloudapi/v5/requests/123456/status")
+	res.ApiResponse.Header.Add("location", "https://api.ionos.com/cloudapi/v6/requests/123456/status")
```

Comment:
not a big deal, but please leave v5 version for `master` branch. This applies also to line 492.

Reply:
Done